sdks/python/apache_beam/io/vcfio.py

```diff
@@ -301,7 +312,14 @@ def next(self):
         return self._convert_to_variant_record(record, self._vcf_reader.infos,
                                                self._vcf_reader.formats)
       except (LookupError, ValueError) as e:
-        raise ValueError('Invalid record in VCF file. Error: %s' % str(e))
+        if self._allow_malformed_records:
+          logging.warning('Invalid VCF record in %s for line %s: %s',
```

Comment:
Ditto.

Reply:
For these ones it would be nice to keep the failing record line. Let me know if the new format with that is good.